sdk/cognitiveservices/textanalytics/Azure.AI.TextAnalytics/src/TextSentimentClass.cs

```diff
@@ -6,7 +6,7 @@ namespace Azure.AI.TextAnalytics
     /// <summary>
     /// </summary>
     #pragma warning disable CA1717 // Only FlagsAttribute enums should have plural names
-    public enum SentimentClass
+    public enum TextSentimentClass
```

Comment:
An enum with the suffix "class"? Perhaps "Classification"? Even if it was a class, this is atypical in the languages we support (it's so...C++ COM 😉).

Reply:
It's standard ML terminology: _In machine learning, the observations are often known as instances, the explanatory variables are termed features (grouped into a feature vector), and the possible categories to be predicted are classes_ per the wikipedia article on it [here](https://en.wikipedia.org/wiki/Statistical_classification).  I realize we have a heavily overloaded term with the meaning of class in programming languages, but I'm hesitant to use the wrong term in an ML library for people who are familiar with these concepts.